### ISSUE TITLE
Feast UI in remote instance

### ISSUE THREAD

**Issue by arumugaguru-mindtree** [feast-dev/feast#2914]
Does web UI work fine if feast is installed in a remote instance? I face the below error on running 'feast ui'. 
```
Error Loading Project Configs
There was an error loading the Project Configurations. Please check that feature_store.yaml file is available and well-formed.
```

However the sample project which is in feast/ui works fine on npm start. 

Please could you help or let me know what I am missing?

**Comment by achals** [comment]
hey @arumugaguru-mindtree we need more context before we can debug this - what do you mean by `feast is installed in a remote instance`?

**Comment by arumugaguru-mindtree** [comment]
Feast is installed in GCP and I access the server using ssh. 

Earlier the request for registry.json was pointing to http://0.0.0.0:888/registry.json. But this was resolved with the 0.22 version. After upgrading to 0.22, I believed this should work fine now. But still the above error is seen. In the web UI console(chrome), I get the Zod error:

````
ZodError: [
  {
    "code": "invalid_enum_value",
    "options": [
      "FLOAT",
      "INT64",
      "STRING",
      "BOOL",
      "BYTES",
      "INT32",
      "DOUBLE",
      "UNIX_TIMESTAMP",
      "INVALID",
      "BYTES_LIST",
      "STRING_LIST",
      "INT32_LIST",
      "INT64_LIST",
      "DOUBLE_LIST",
      "FLOAT_LIST",
      "BOOL_LIST",
      "UNIX_TIMESTAMP_LIST",
      "NULL"
    ],
    "path": [
      "entities",
      0,
      "spec",
      "valueType"
    ],
    "message": "Invalid enum value. Expected 'FLOAT' | 'INT64' | 'STRING' | 'BOOL' | 'BYTES' | 'INT32' | 'DOUBLE' | 'UNIX_TIMESTAMP' | 'INVALID' | 'BYTES_LIST' | 'STRING_LIST' | 'INT32_LIST' | 'INT64_LIST' | 'DOUBLE_LIST' | 'FLOAT_LIST' | 'BOOL_LIST' | 'UNIX_TIMESTAMP_LIST' | 'NULL'"
  },
  {
    "code": "invalid_enum_value",
    "options": [
      "FLOAT",
      "INT64",
      "STRING",
      "BOOL",
      "BYTES",
      "INT32",
      "DOUBLE",
      "UNIX_TIMESTAMP",
      "INVALID",
      "BYTES_LIST",
      "STRING_LIST",
      "INT32_LIST",
      "INT64_LIST",
      "DOUBLE_LIST",
      "FLOAT_LIST",
      "BOOL_LIST",
      "UNIX_TIMESTAMP_LIST",
      "NULL"
    ],
    "path": [
      "entities",
      1,
      "spec",
      "valueType"
    ],
    "message": "Invalid enum value. Expected 'FLOAT' | 'INT64' | 'STRING' | 'BOOL' | 'BYTES' | 'INT32' | 'DOUBLE' | 'UNIX_TIMESTAMP' | 'INVALID' | 'BYTES_LIST' | 'STRING_LIST' | 'INT32_LIST' | 'INT64_LIST' | 'DOUBLE_LIST' | 'FLOAT_LIST' | 'BOOL_LIST' | 'UNIX_TIMESTAMP_LIST' | 'NULL'"
  }
]
    at yk (index.mjs:449:23)
    at n.value (index.mjs:561:16)
    at n.value (index.mjs:541:29)
    at useLoadRegistry.ts:35:47
```
The feature repo that I use is the sample one that is created by default using feast init. No other changes are done to the repo.

I hope this info helps.

**Comment by adchia** [comment]
this is super helpful. will take a look asap!

**Comment by arumugaguru-mindtree** [comment]
Hi,

I figured out that the entity "dummy id" which was created for entityless feature view was causing the issue. The value type was not passed while defining the "dummy id" entity in featureview.py. Also, as per my understanding this should be used only when a feature view is created without an entity. However it gets appended even if a feature view is created with an entity.

**Comment by adchia** [comment]
Should be fixed now I think in Feast 0.22.1!
